### DISCUSSION TITLE
Error when copying only some story maps

### DISCUSSION THREAD

**Discussion by FarkleberrySlim** [EsriPS/arcgis-assistant-feedback#76] [Q&A]
I am trying to copy five story maps over from one account to another. I have successfully copied three story maps but two of them repeatedly give me the error below:

![image](https://github.com/EsriPS/arcgis-assistant-feedback/assets/144733502/f9102054-c426-4445-964a-3c31b012ba45)

I have also copied over a collection that contains all 5 story maps but I would still like to be able to copy those remaining two individually. Any ideas as to why some of these worked and others did not would be much appreciated. A solution would be even better!

Best,
Chris

**Comment by oevans** [comment]
@FarkleberrySlim -- Sorry you are experiencing an issue copying some of your stories. I'm not sure what might cause this, so I have a few questions that might help us figure out what's going on:

* Is there anything different you notice about the stories that won't copy? Are they drafts vs. published stories? Any weird characters in the titles? Any specific content in some vs. the others?
* Are the stories in your own account?

Also, FYI, if you copy a collection, it will be referencing the original stories, not the duplicates. You'll either need to manually edit the JSON to replace the old storyIDs with the new ones, or it might be easier to just recreate the collection with the new stories.

**Reply by FarkleberrySlim** [reply]
@oevans Thanks for the quick reply. 

- They are actually the two smallest and least complex of the story maps. One contains the major heading, some introductory text and a map. The other contains a bit more text, images, and maps. They are both published but see image and text below. The titles have colons (but so do the three that did copy over). As a quick test, I just tried taking the colon out of the title and copying one and it still did not work.
- The stories are in my own account.

I just tried editing and publishing one of the two story maps and got these errors when hitting publish:
![image](https://github.com/EsriPS/arcgis-assistant-feedback/assets/144733502/3a967598-5df8-4b2b-8b69-ad505371ceb8)

Thanks for the info on the collections. I noticed that changes I made to a story map in one account were reflected in the collection transferred to the other account. I'll be sure to make a new one when I can get the last two transferred over.

**Reply by oevans** [reply]
Are these stories public (or can you make them public temporarily)? If so, please share the story link or ID. Otherwise, I'm not sure we can do much additional troubleshooting unless we can take a look at the story data on our end.

**Reply by FarkleberrySlim** [reply]
Here they are:

https://storymaps.arcgis.com/stories/08addd3f9dc945618bab08f557df7e83
https://storymaps.arcgis.com/stories/24e95b0bbb074d84ad23d3a794cc01e4

**Reply by oevans** [reply]
Hi and I apologize for the delay in my response. Did you ever get these stories copied over? The only thing we noticed is that a web map in both of these stories has a huge number of layers (over 120!). I'm not sure what specific issue that might cause in copying the story using ArcGIS Assistant, but it certainly jumped out at us as something way outside the range of normal. Was that web map also used in any of the other stories that copied successfully?

**Reply by FarkleberrySlim** [reply]
I never figured out how to transfer those but others using the same web map were transferred successfully. Luckily, they were small, not super complex, and able to be relatively easily recreated on the other account.
